### ISSUE TITLE
panoptes job naming

### ISSUE THREAD

**Issue by nick-youngblut** [snakemake/snakemake#1209]
It appears that the user cannot set the workflow name for panoptes via `--wms-monitor-arg`, at least, I haven't been able to figure out how or see any documentation on it. The snakemake-generated UUID names do not work well when viewing a bunch of workflows in panoptes, and manually renaming them is a big enough hassle that it is generally not worth the benefit gained from panoptes.

So, it would be great if the user could automatically set the workflow name via `--wms-monitor-arg`. If this is already possible, it would be great to have more docs on that.

**Comment by fgvieira** [comment]
It seems PR #2444 tries to fix this.

**Comment by weber8thomas** [comment]
> It seems PR #2444 tries to fix this.

Yes absolutely! I'm waiting for the new version Johannes is working on (where the different interfaces executor plugins are being reshaped) to finalise and adapt the code but the code in my PR is already working with panoptes on my instance :)

**Comment by fgvieira** [comment]
It seems `logging` will stay in the main `snakemake` repo (at least I could not find any repos for logging).
@johanneskoester can you confirm? If so, then there is no need to wait and we can process with this PR, no?

**Comment by fgvieira** [comment]
It seems the fix has just been merged.
@nick-youngblut, can you give it a try and see if it works?

**Comment by kjohnsen** [comment]
I tried this and it worked
